assets/RNAseq_report.Rmd

```diff
@@ -305,6 +307,7 @@ The raw count table and normalized count tables are available [here](./different
 The differential expression analysis is performed using the raw gene count table. 
 For PCA analysis and heatmap plotting, the regularized logarithm (rlog) normalized gene counts were employed.
 The vst (variant stabilizing transformation) normalized counts are also provided.
+The number of subset genes for the transformation is `r nsub_genes_text`.
```

Comment:
```suggestion
The number of subset genes for the `vst` transformation is `r nsub_genes_text`.
```

Reply:
Thanks Gisela, I've made the text change to RNAseq_report.Rmd